app/controllers/pdl.py

```diff
@@ -11,6 +11,9 @@
 from app.models.allprograms import AllProgram
 
 
+LANDING_PAGE_DATA_PATH = os.path.join("controllers", "data", "landingpage")
+ALLPROGRAM_DATA_JSON = "allPrograms.json"
```

Comment:
This must be changed to "allprograms.json" to match the current file name. 
On a related note, separating words in filenames by underscores can give better readability.

Reply:
updated. I agree. All the file names used in here are the same ones as the filenames in box folder. I also prefer using underscore.

Reply:
Thanks.